src/nibpkg/refmers.nim

```diff
@@ -1,23 +1,65 @@
 import hts
 import kmers
 import tables
+import svidx
 
-proc countRefKmers*(input_fn: string):CountTable[Bin] =
+type
+    Chunk* = object
+        chrom_name: string
+        chrom_start: int
+        chrom_end: int
+    
+proc addRefCount*(refKmers: CountTableRef[uint64], svKmers: svIdx) =
+    for k, v in pairs(refKmers): 
+        if svKmers.hasKey(k):
+            svKmers[k].refCount += uint32(v)
+
+proc createChunk*(fai: Fai, chunk_size: int): seq[Chunk] =
+    var chunkSeq = newSeq[Chunk]()
+    for i in 0..<fai.len:
+        let chrom_name = fai[i]
+        let chrom_len = fai.chrom_len(chrom_name)
+        for j in countup(0, chrom_len, chunk_size):
+            var chunk: Chunk
```

Comment:
here, you can replace following 5 lines with:
`yield Chunk(chrom_name:chrom_name, chrom_start:j, chrom_end: j + chunk_size)`